### PR TITLE
DOCS-15884 1.2 release notes

### PR DESCRIPTION
**Pull request by kennethdyer**
[Jira](https://jira.mongodb.org/browse/DOCS-15884)

[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCS-15884-1.2-release-notes/release-notes/1.2/)

[Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=63ebc7c2dcefe977976f244c)

### CODE REVIEW

File: source/release-notes/1.2.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+-------------------------------------
+
+{+c2c-product-name+} 1.2 adds support for sync between clusters running 
+older versions of MongoDB.
+
+Sync between MongoDB 5.0, 4.4, and 4.2 clusters replicate CRUD operations only.  
+
+``mongosync`` **cannot** detect DDL operations in pre-6.0 clusters.
+DDL operations, such as creating and dropping databases and collections,
+adding indexes, etc. during replication can cause the destination cluster
```
Comment (jmd-mongo) [inline source/release-notes/1.2.txt:29]: [suggestion]

Consider replacing "etc." to be more inline with style guide: https://www.mongodb.com/docs/meta/style-guide/terminology/general-term-guidelines/avoid-obscure-words/

File: source/release-notes/1.2.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+to fall out of sync with the source.
+
+For more information, contact your account team or 
+`MongoDB <https://www.mongodb.com/contact>`__.
+
+
+Guardrails for Unsupported Versions of MongoDB
+----------------------------------------------
+
+``mongosync`` now returns an error if the user calls :ref:`c2c-api-start`
```
Comment (jmd-mongo) [inline source/release-notes/1.2.txt:39]: [suggestion]

Consider "returns an error if you call...".

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the updates, @kennethdyer. I've left some suggestions for your consideration.

Thanks!
Joe

File: source/release-notes/1.2.txt (hunk 1 above)
Comment (kennethdyer) [reply source/release-notes/1.2.txt:29]: Mitigated

File: source/release-notes/1.2.txt (hunk 2 above)
Comment (kennethdyer) [reply source/release-notes/1.2.txt:39]: Fixed

**Review by jmd-mongo — APPROVED** [review]
Thanks for the updates, @kennethdyer. This one LGTM at this point.

Thanks!
Joe

File: source/release-notes/1.2.txt [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+If you would like to use ``mongosync`` with an older version of MongoDB,
+contact Professional Services.
+
+
+Guardrails for Unsupported Versions of MongoDB
+----------------------------------------------
+
+``mongosync`` now returns an error if you call :ref:`c2c-api-start`
+to sync clusters of an unsupported release of MongoDB Server.
```
Comment (autarch) [inline source/release-notes/1.2.txt:33]: I think it would be clearer to say "running an unsupported" rather than "of an unsupported".

**Review by autarch — COMMENTED** [review]
LGTM % one very small language nit.

File: source/release-notes/1.2.txt (hunk 3 above)
Comment (kennethdyer) [reply source/release-notes/1.2.txt]: Fixed

File: source/release-notes/1.2.txt [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   ``mongosync`` 1.2 Released February 16, 2023
+
+Support for Older Versions of MongoDB
```
[the remark this answers is not in the document]
Comment (kennethdyer) [reply source/release-notes/1.2.txt:19]: Fixed

File: source/release-notes/1.2.txt [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+-------------------------------------
+
+{+c2c-product-name+} 1.2 can sync between clusters running 
+older (pre-6.0) releases of MongoDB Server.  Use of this feature
+requires additional preparation and configuration.
+
+Contact Professional Services if you would like to sync older
+versions of MongoDB.
```
[the remark this answers is not in the document]
Comment (kennethdyer) [reply source/release-notes/1.2.txt:27]: Fixed

**Review by alan-zheng-mdb — APPROVED** [review]
LGTM

**Pull request merged**